public/js/dashboard-application.js

```diff
@@ -26,33 +26,53 @@ function getAndSendApplicationData(toggleHasApplication) {
 	if (school_entry == "Other") {
 		school_entry = $('input[name="school_other"]').val()
 	}
-
 	if (school_entry == "") {
 		alert("Please specify your school.")
 		return false
 	}
 
 	// get gender
-	let gender_entry = $('select[name="gender"]')
+	// let gender_entry = $('select[name="gender"]')
+	// 	.find(":selected")
+	// 	.text()
+	// if (gender_entry == "Other") {
+	// 	gender_entry = $('input[name="gender_other"]').val()
+	// }
+
+	// if (gender_entry == "") {
+	// 	alert("Please specify your self-identified gender.")
+	// 	return false
+	// }
+
+	//NEW LOGIC
```

Comment:
Could you update the comment to something more descriptive (ie. Logic for remote hackathon kit selections)